.eslintignore

```diff
@@ -1,2 +1,4 @@
 dist/**
-.github/**
+yarn.lock
+demos/**
```

Comment:
```suggestion
demos/**
!demos/*.html
```

This will make the `eslintignore` compatible with #2531

Reply:
Maybe remove `demos/**` from `eslintignore? Only html files in demos directory.

Reply:
Sure! 👍🏾